### ISSUE TITLE
Shifting runtext16.ppm with different position

### ISSUE THREAD

**Issue by Gadag007** [hzeller/rpi-rgb-led-matrix#228]
Hi, 
I am using hub75(32x32),1:16 Multiplex , I am trying to shift down runtext16.ppm, meaning I want to start it from 16th row or any row for that matter.  But i am not able to do that , please any one suggest me, I just need few hints on where to get started.

Thanks

**Comment by hzeller** [comment]
You are not very descriptive what you need, but [this is the place](https://github.com/hzeller/rpi-rgb-led-matrix/blob/master/examples-api-use/demo-main.cc#L281) where the image scroller example does its things.

However, this is really a simple hacky example and not really a good place to add extensions. You should really just read that code, get inspired and write a fresh program what you'd like to do. You can get inspired by the [led-image-viewer](https://github.com/hzeller/rpi-rgb-led-matrix/blob/master/utils/led-image-viewer.cc) or the python example [scrolling an image](https://github.com/hzeller/rpi-rgb-led-matrix/blob/master/python/samples/image-scroller.py).

**Comment by Gadag007** [comment]
Hi hzeller, I have updated my query.

**Comment by Gadag007** [comment]
Hi Maddogs, Thanks for your response, But I'm working on C++ APIs.

**Comment by hzeller** [comment]
So if you want to shift the image vertically, why don't you just add a constant (e.g. 16) to `y` when call `SetPixel()` @Gadag007 ?

**Comment by Gadag007** [comment]
Thanks so much hzeller, you rock... I was able to achieve the shift steps. 

Also, I had to spawn a new thread with imageScroller class replicated to make parallel ppm at once. I will make it more readable and post it below so that it can help someone else.

Lastly, I wanted to run the ppm and stop it once it hits the left most column in the matrix, probably by changing scroll_ms to 0. I could go with text-example but, I want to achieve this in ppm.

 Any suggestions?

Thanks again.
